resume-buddy/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -33,14 +35,17 @@
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet {
 
+  private static UserType userType;
```

Comment:
Why is this a static variable?

Reply:
If I change it to be non-static I get this error: non-static variable userType cannot be referenced from a static context